compose/config/validation.py

```diff
@@ -216,15 +234,17 @@ def _parse_valid_types_from_schema(schema):
 
 def validate_against_fields_schema(config):
     schema_filename = "fields_schema.json"
-    return _validate_against_schema(config, schema_filename)
+    format_checkers = ["ports", "environment"]
+    return _validate_against_schema(config, schema_filename, format_checkers)
 
 
 def validate_against_service_schema(config, service_name):
     schema_filename = "service_schema.json"
-    return _validate_against_schema(config, schema_filename, service_name)
+    format_checkers = ["ports"]
```

Comment:
Cool, maybe an inline comment for that so we remember why these lists are different?